cli/cmd/auth.go

```diff
@@ -148,6 +148,13 @@ func init() {
 }
 
 func verifyAuthParams(authParams *authCmdParams) error {
+
+	const parametersRequiredMessage = "keptn auth requires api-token and endpoint \n\n" +
+		"For more information on how to obtain token and endpoint to go https://keptn.sh/docs/%s/reference/cli/#authenticate-keptn-cli \n\n" +
+		"Alternatively,To quickly access Keptn, you can use a port-forward and then authenticate your Keptn CLI: \n" +
+		"- kubectl -n keptn port-forward service/api-gateway-nginx 8080:80 \n" +
```

Comment:
Is it possible to make the namespace a variable here?

Reply:
By making the namespace variable, my understanding it that we want fetch and show the namespace that keptn is installed in (passed in args of `keptn install`)?
If that is the case, we will have to write a separate function that detects the installation namespace.

But in my opinion, whosoever follows the tutorial, they will probably have the default installation namespace as `keptn`.

Do let me know your thoughts!

Reply:
I agree that most people will follow the tutorial where you install it in the `keptn` namespace, nevertheless, I think the variable `namespace` is available (see line `namespace, err = smartKeptnCLIAuth()`), so we could make use of that (and in case it is empty or an has occured when fetching it, it could default to `keptn`).

So my proposal would be:
```
namespace := "keptn"
namespace, err = smartKeptnCLIAuth()
...
return fmt.Errorf(err.Error()+parametersRequiredMessage, keptnReleaseDocsURL, namespace)
```